### PR TITLE
Adding a feedback form in k8s doc

### PR DESCRIPTION
**Pull request by macolso**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [ ] The `title, `template`, and `date` are all set
- [ ] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [ ] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [ ] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [ ] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [ ] Headings are using Title Case
- [ ] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [ ] Have tested with `npm run test` and resolved all errors
- [ ] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

**Review by karthik2804 — APPROVED** [review]

File: content/spin/v1/kubernetes.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v1/kubernetes
 
 In addition to `spin up` Fermyon also offers Fermyon Cloud to deploy spin apps into production, so why use Spin with Kubernetes? For users that have made existing investments into Kubernetes or have requirements that their applications stay within certain clouds, not be on shared infrastructure, or run on-premise, Kubernetes provides a robust solution.
 
-> Fermyon wants to know how we can build you the best Spin experience on Kuberenetes. Let us know what you need to effeciently run Spin applications at scale on Kubernetes with this [short survey](https://fibsu0jcu2g.typeform.com/to/H8G8Nzgs?utm_campaign=thanks&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-_IwqD36cFYu9skvZBobOVZjjdXtGO3Z126WweaeKBWZz8wM4G_LZG7L0esIOS4tcASX8LM).  
+> Fermyon wants to know how we can build you the best Spin experience on Kuberenetes. Let us know what you need to efficiently run Spin applications at scale on Kubernetes with this [short survey](https://fibsu0jcu2g.typeform.com/to/H8G8Nzgs?utm_campaign=thanks&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-_IwqD36cFYu9skvZBobOVZjjdXtGO3Z126WweaeKBWZz8wM4G_LZG7L0esIOS4tcASX8LM).
```
Comment (itowlson) [inline content/spin/v1/kubernetes.md:49]:
```suggestion
> Fermyon wants to know how we can build you the best Spin experience on Kubernetes. Let us know what you need to efficiently run Spin applications at scale on Kubernetes with this [short survey](https://fibsu0jcu2g.typeform.com/to/H8G8Nzgs?utm_campaign=thanks&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-_IwqD36cFYu9skvZBobOVZjjdXtGO3Z126WweaeKBWZz8wM4G_LZG7L0esIOS4tcASX8LM).  
```

**Review by itowlson — CHANGES_REQUESTED** [review]
I'm very sorry but there was a _second_ typo which I missed when looking at it before

**Review by itowlson — APPROVED** [review]

File: content/spin/v2/kubernetes.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/kubernetes
 
 In addition to `spin up` Fermyon also offers Fermyon Cloud to deploy spin apps into production, so why use Spin with Kubernetes? For users that have made existing investments into Kubernetes or have requirements that their applications stay within certain clouds, not be on shared infrastructure, or run on-premise, Kubernetes provides a robust solution.
 
+> Fermyon wants to know how we can build you the best Spin experience on Kuberenetes. Let us know what you need to efficiently run Spin applications at scale on Kubernetes with this [short survey](https://fibsu0jcu2g.typeform.com/to/H8G8Nzgs?utm_campaign=thanks&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-_IwqD36cFYu9skvZBobOVZjjdXtGO3Z126WweaeKBWZz8wM4G_LZG7L0esIOS4tcASX8LM).
```
Comment (tpmccallum) [inline content/spin/v2/kubernetes.md:48]:
```suggestion
> Fermyon wants to know how we can build you the best Spin experience on Kubernetes. Let us know what you need to efficiently run Spin applications at scale on Kubernetes with this [short survey](https://fibsu0jcu2g.typeform.com/to/H8G8Nzgs?utm_campaign=thanks&utm_source=hs_email&utm_medium=email&_hsenc=p2ANqtz-_IwqD36cFYu9skvZBobOVZjjdXtGO3Z126WweaeKBWZz8wM4G_LZG7L0esIOS4tcASX8LM).  
```

**Review by tpmccallum — APPROVED** [review]
Kubernetes misspelled on the `v2` page—otherwise LGTM.

**Pull request merged**